source/documentation/managing_apps/autoscaling.erb

```diff
@@ -0,0 +1,288 @@
+## Autoscaling
+
+Apps can be configured to dynamically scale in and out using autoscaling
```

Comment:
In and out rather than up and down?

Reply:
I associate scaling up/down with vertical scaling (increase CPU/memory/network provisioning) and scaling in/out with horizontal scaling

Reply:
Fair enough. Any user research on how users of the PaaS would refer to it too?

Reply:
Scaling up/down certainly feels more natural language than scaling in/out

Reply:
Since in/out and up/down _do_ have meaning in the industry, to avoid confusion maybe avoid the terms entirely.

> Apps can be configured to dynamically increase or decrease the number of running instances using autoscaling.

Reply:
good suggestion, resolved in 13d1ec2b4d78398383da0b2cb4d4ea7cd80a0739